src/config/fenix.js

```diff
@@ -35,6 +38,7 @@ export default {
         { key: 'build_id', label: 'Build ID' },
         { key: 'version', label: 'Major Version' },
       ],
+      defaultValue: 'build_id',
     },
   },
```

Comment:
The backend now requires `app_id` be provided. It should be one of:
* `org_mozilla_fenix` for Fenix Nightly
* `org_mozilla_firefox_beta` for Fenix Beta
* `org_mozilla_firefox` for Fenix Release